.eslintrc

```diff
@@ -15,5 +15,6 @@
     "Box2D": false,
     "astar": false,
     "Graph": false,
+    "PIXI": false,
```

Comment:
Global variables for eslint. We never declare these variables because they're accessible through the tagpro API and such, this is telling eslint that it's okay that we're not declaring them.